Userland/Libraries/LibWeb/HTML/HTMLMediaElement.cpp

```diff
@@ -75,10 +75,11 @@ Bindings::CanPlayTypeResult HTMLMediaElement::can_play_type(DeprecatedString con
     // Generally, a user agent should never return "probably" for a type that allows the codecs parameter if that parameter is not present.
     if (type == "application/octet-stream"sv)
         return Bindings::CanPlayTypeResult::Empty;
-    // FIXME: Eventually we should return `Maybe` here, but for now `Empty` is our best bet :^)
-    //        Being honest here leads to some apps and frameworks skipping things like audio loading,
-    //        which for the time being would create more issues than it solves - e.g. endless waiting
-    //        for audio that will never load.
+
+    if (type.contains("video/webm"sv))
+        return Bindings::CanPlayTypeResult::Probably;
+    if (type.contains("video/"sv))
+        return Bindings::CanPlayTypeResult::Maybe;
```

Comment:
This needs to use proper mime type parsing and compare against the full essence, not just a contains check (which incorrectly matches `<garbage>video/webm<garbage>` and `other/type;parameter="video/webm"`)